server/gitrest/packages/gitrest-base/src/routes/git/blobs.ts

```diff
@@ -22,6 +22,7 @@ export function create(
     repoManagerFactory: IRepositoryManagerFactory): Router {
     const router: Router = Router();
     const repoPerDocEnabled: boolean = store.get("git:repoPerDocEnabled") ?? false;
+    // eslint-disable-next-line @typescript-eslint/no-misused-promises
```

Comment:
Was this rule triggered because of any update to eslint packages like `"@fluidframework/eslint-config-fluid"`? That package did not seem to have been updated in package.json so I was curious.

Reply:
I think it's because of the express type updates. https://github.com/microsoft/FluidFramework/pull/14300/files#diff-627b534b7a24752905b714ad5d72715eeb61770e3165d7fc6d23f7b063778962

I'm not sure how it was working before, tbh.